### ISSUE TITLE
`tabindex` is currently global attribute (SCR20)

### ISSUE THREAD

**Issue by momdo** [w3c/wcag#284]
https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/SCR20#SCR20-ex2

> Note: This example uses tabindex on an img element. Even though this is currently invalid, it is provided as a transitional technique to make this function work. 

tabindex attribute is defined as <a href="https://www.w3.org/TR/html51/dom.html#sec-global-attributes">global attribute</a> in HTML5. So this example is valid.

**Comment by fstrr** [comment]
[Example two on the referenced WCAG 2.0 page](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/SCR20#SCR20-ex2) (and [WCAG 2.1 page](https://www.w3.org/WAI/WCAG21/Techniques/client-side-script/SCR20)) could do with updating or removing. I don't think we should be encouraging people to make images into controls.

This would be better as an ARIA link:

```html
<span onclick="nextPage();" onkeypress="nextPage();" role="link" tabindex="0">
  <img alt="Go to next page" src="arrow.gif">
</span>
```
